shm/cargo.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <iostream>
+#include <string>
+
+class Cargo {
+private:
+    const std::string name_;
+    size_t amount_;
+    size_t basePrice_;
+
+public:
+    Cargo(const std::string name, size_t amount, size_t basePrice)
```

Comment:
The definition should be in .cpp files. Only simple getter/setters are allowed in .hpp